templates/single-quiz.php

```diff
@@ -40,11 +42,13 @@
 			<?php
 
 			/**
-			 * Action inside before the question content on single-quiz page
+			 * Action inside before the question content on single-quiz page.
 			 *
-			 * @hooked Sensei_Quiz::the_user_status_message  - 10
+			 * @hook sensei_single_quiz_questions_before
+			 *
+			 * @param {int} $the_quiz_id The quiz ID.
```

Comment:
```suggestion
			 * @param {int} $quiz_id The quiz ID.
```

Same reason as the other comments.